### DISCUSSION TITLE
Problem with stirring blade components with Livewire components.

### DISCUSSION THREAD

**Discussion by Xsaven** [livewire/livewire#4409] [7. Bugs (archived)]
The situation is as follows.
I use the Livewire component as a page, for this, my next design is indicated in the routes:
```
Route::get('/', \ComponentNamespace\AppContainerComponent::class);
```
in `ServiceProvider`
```
\Livewire::component('app-container', \ComponentNamespace\AppContainerComponent::class)
```
And when I am trying inside the template of this component, insert the other component of Livewire, it gives me an error:
```
BadMethodCallException
Method Illuminate\View\View::childHasBeenRendered does not exist. (View: /.../resources/views/widgets/main_sidebar.blade.php)
```
I sold a place where an error occurs, it turned out that in the variable `$_instance` is an instance of the `Illuminate\View\View` class:
![image](https://user-images.githubusercontent.com/1726771/146595408-f31e11fb-3210-4e57-b949-d412a5559b37.png)
And in the end, we can get something in the template variable and you will not give a check you do not:
![image](https://user-images.githubusercontent.com/1726771/146595645-d4567d0b-9f6c-417e-88e6-1960d447d6bd.png)

Fix please!

P.S.
So that, at the beginning of each template, where I use a Livevier component, I put the following design:
```
@php($_instance = $_instance instanceof \Illuminate\View\View ? null : $_instance)
```

PHP: v8.0
Laravel: v8.76.1
Livewire: v2.8.2

**Comment by Xsaven** [comment]
Clearly, the developer of the asshole, it is more important to what list I wrote, and not the decision of the bug!

**Reply by joshhanley** [reply]
@Xsaven please read my comment again. We're no longer using GitHub issues to track bugs, instead we are using discussions. That is a decision by Caleb who is the owner of this project and I referenced the announcement of that decision in the comment, so please have a look at that.

You had already opened this discussion and that is why that issue was closed. If you hadn't we'd have converted your issue to a discussion so it can be investigated.

Bug reports are important to us, and they will be investigated when we have time.

But using language like that is not acceptable in this repo regardless of what has happened.

I'd appreciate if you at least try to have a civil discussion with us if you aren't happy with how we are handling things.

Thank you.

**Reply by calebporzio** [reply]
Yep, thanks for being so kind @joshhanley and sorry you had to endure this person and their insults.

Locking this discussion as this kind of behavior won't be tolerated in this repo.
